protocol/daemons/pricefeed/client/price_encoder/price_encoder_test.go

```diff
@@ -272,3 +284,135 @@ func TestConvertPriceUpdate_Mixed(t *testing.T) {
 		})
 	}
 }
+
+func TestUpdatePrice_Failure(t *testing.T) {
```

Comment:
Improve price_encoder coverage.

Reply:
nit: maybe a comment here like: failure within grace period is an Info log, after grace period should be Error log

Reply:
Updated the inline comment below!